views/js/pciCreator/ims/mathEntryInteraction/creator/tpl/addAlternativeBtn.tpl

```diff
@@ -0,0 +1,3 @@
+<button class="math-entry-response-correct btn-info">
+    <i class="adder"> </i> {{__ "Add alternative"}}
```

Comment:
are you sure this is the correct form for using the translated string

Reply:
It is the same we use in other templates